pacai/student/qlearningAgents.py

```diff
@@ -23,109 +23,60 @@ class QLearningAgent(ReinforcementAgent):
-        """
-        Returns Q(state,action)
+    - def getQValue(self, state, action):
+        - Returns Q(state,action)
         Should return 0.0 if we never seen
         a state or (state,action) tuple
-        """
-
-        """
-        Description:
-        [Enter a description of what you did here.]
-        """
 
-        """ YOUR CODE HERE """
-        util.raiseNotDefined()
-        """ END CODE """
-
-    def getValue(self, state):
-        """
-        Returns max_action Q(state,action)
+    - def getValue(self, state):
+        - Returns max_action Q(state,action)
         where the max is over legal actions. Note that if
         there are no legal actions, which is the case at the
         terminal state, you should return a value of 0.0.
-        """
 
-        """
-        Description:
-        [Enter a description of what you did here.]
-        """
-
-        """ YOUR CODE HERE """
-        util.raiseNotDefined()
-        """ END CODE """
-
-    def getPolicy(self, state):
-        """
-        Compute the best action to take in a state. Note that if there
+    - def getPolicy(self, state):
+        - Compute the best action to take in a state. Note that if there
         are no legal actions, which is the case at the terminal state,
         you should return None.
-        """
-
-        """
-        Description:
-        [Enter a description of what you did here.]
-        """
 
-        """ YOUR CODE HERE """
-        util.raiseNotDefined()
-        """ END CODE """
+    - def getAction(self, state):
 
-    def getAction(self, state):
-        """
-        Compute the action to take in the current state. With
+        -         Compute the action to take in the current state. With
         probability self.epsilon, we should take a random action and
         take the best policy action otherwise. Note that if there are
         no legal actions, which is the case at the terminal state, you
         should choose None as the action.
 
         HINT: You might want to use util.flipCoin(prob)
         HINT: To pick randomly from a list, use random.choice(list)
-        """
 
         # Pick Action
         legalActions = self.getLegalActions(state)
         action = None
-
-        """
-        Description:
-        [Enter a description of what you did here.]
-        """
-
-        """ YOUR CODE HERE """
-        util.raiseNotDefined()
-        """ END CODE """
+        
 
         return action
 
-    def update(self, state, action, nextState, reward):
-        """
-        The parent class calls this to observe a
+    - def update(self, state, action, nextState, reward):
+        - The parent class calls this to observe a
         state = action => nextState and reward transition.
         You should do your Q-Value update here
 
         NOTE: You should never call this function,
         it will be called on your behalf
-        """
+    """
 
+    def __init__(self, index, **args):
         """
-        Description:
```

Comment:
Put back the Description portion ( i believe this was an area for the students to put in comments)